### ISSUE TITLE
Error 559

### ISSUE THREAD

**Issue by damiendep** [arichorn/CercubePlusExtra#4]
It looks like the Dev of "C e r c u b e" did something on his server since downloading YT videos returns an Error 559.
I tried with the latest build available here [v17.49.4-5.3.12] on different devices and I get the same error every time. In some cases after a couple of days only. Reverting to earlier build doesn't fix that issue, therefore I assume it's on the server side. Every other features seam to work normally.
Anyone else experiencing the same?
![IMG_0653](https://user-images.githubusercontent.com/120914779/208291687-fc0b184b-3fa4-47ea-9dc1-f492f4cc2857.png)

**Comment by cxen** [comment]
Same issue since today.

**Comment by damiendep** [comment]
Other forks give the same Error 559, it's definitely because of the dev.

**Comment by arichorn** [comment]
I don’t know what the cause this is coming from but I’m getting the same error just like y’all. Contact Cercube Dev.

**Comment by arichorn** [comment]
I went on Reddit and found out Cercube is no longer being updated at least that’s what I think but hold on you can still use Cercube but I will add something to CercubePlus.

**Comment by arichorn** [comment]
https://github.com/arichorn/CercubePlusExtra/releases/tag/v17.49.6-5.3.11-(47)
use 17.49.6-5.3.11
Cercube has been downgraded and this should also forcefully enable the hideCercubeDownload button but without any option to disable it. I did it like this because you will get the Error 559 if you attempted to download stuff there 😭

**Comment by damiendep** [comment]
Thank you for that solution arichorn

**Comment by vadim-a-yegorov** [comment]
Best solution ever 👍

**Comment by arichorn** [comment]
use v18.01.2 I’ve fixed it so it’ll properly work and properly hide the download button in the tab bar

**Comment by vadim-a-yegorov** [comment]
Yeah if button doesn’t work better remove it than fix, I appreciate

**Comment by arichorn** [comment]
Also anyone using 18.01.2. Does anything work fine, I have no clue if all the options worked in settings
or any of the new Localization I’ve added in this version, tell me if any feature works because I’m unable to ‘CercubePlus‘ while I’m using ‘uYouPlusExtra‘

**Comment by dayanch96** [comment]
same as in https://github.com/arichorn/CercubePlusExtra/issues/6#issuecomment-1416870152
